doc/design/design_model_zoo_pai.md

```diff
@@ -0,0 +1,97 @@
+# Model Zoo on PAI TensorFlow
+
+## Introduction
+
+[Alibaba PAI](https://www.alibabacloud.com/product/machine-learning) is an end-to-end platform that provides various machine learning algorithms to meet users' data mining and analysis requirements. PAI supports TensorFlow and many other machine learning frameworks. SQLFlow model zoo also works on PAI. [Model Zoo Design Doc](./design_model_zoo.md) is a high level design for SQLFlow model zoo. This document is about how to do model training, model prediction, and model analysis using SQLFlow model zoo on PAI.
+
+## Background
+### Submit a Task to PAI
+A PAI program is a normal program in TensorFlow or other framework, except that PAI requires its application expose several command line options to communicate with [MaxCompute](https://www.alibabacloud.com/product/maxcompute) and OSS([Alibaba Cloud Object Storage Service](https://www.alibabacloud.com/product/oss)). Typically, a user submit a PAI task with one of the following methods:
+1. Use the MaxCompute CLI `odpscmd`:
+```bash
+$ odpscmd -e "pai -name tensorflow -Dscript=file:///my/training/script.py -Dother_param=other_param_value..."
+
+```
+When a user runs the command above, the CLI will submit the task to PAI then print a logview URL and an instance ID to stdout, after which the user can safely terminate the CLI. The user can check the logview to get the running status of the task or run:
+```bash
+$ odpscmd -e "wait instanceID"
+```
+to attach to the running task.
+
+
+2. Use the MaxCompute Python Package `pyodps`:
+```python
+
+from odps import ODPS
+
+conn = ODPS(...)
+inst = conn.run_xflow('tensorflow', parameters={'script':      'file:///my/training/script.py',
+               	                                'other_param': 'other_param_value' ... })
+print(instance.get_logview_address())
+```
+This code snippet asynchronously submits a task to PAI and prints the logview URL to stdout.
+
+### Train/Predict/Analyze in SQLFlow with Model Zoo
+In SQLFlow, we train a model with the following SQLFlow extened SELECT statement:
```

Comment:
we train a model => we train/predict/analyze a model ...

Reply:
Done.